deploy/dev/values.yaml

```diff
@@ -37,7 +27,7 @@ ingress:
           paths:
           - backend:
               serviceName: "kiva-ui"
-              servicePort: 443
+              servicePort: *servicePort
```

Comment:
Likewise here I prefer to use port aliases, so if we named the service port as `service.portName` we could likewise use that alias here.